tests/func/experiments/test_remove.py

```diff
@@ -0,0 +1,49 @@
+from funcy import first
+
+from dvc.repo.experiments.utils import exp_refs_by_rev
+
+
+def test_remove_experiments_by_ref(tmp_dir, scm, dvc, exp_stage, caplog):
+    results = dvc.experiments.run(exp_stage.addressing, params=["foo=2"])
+    exp = first(results)
+    ref_info = first(exp_refs_by_rev(scm, exp))
+
```

Comment:
Thanks for expanding on these! As long as we are adding more tests and conditions, what do you think about adding another experiment and asserting that it doesn't get removed? What about testing multiple refs passed at once (`dvc exp remove abc123 def456`)?